### PR TITLE
Update swagger

### PR DESCRIPTION
**Pull request by zolvarga**
# Checklist
- [ ] I have read the [contribution guidelines] (https://github.com/Azure/azure-iot-sdk-node/blob/master/.github/CONTRIBUTING.md).
- [ ] I added or modified the existing tests to cover the change (we do not allow our test coverage to go down).
- If this is a modification that impacts the behavior of a public API
  - [ ] I edited the corresponding document in the `devdoc` folder and added or modified requirements.

# Reference/Link to the issue solved with this PR (if any)

# Description of the problem

# Description of the solution

### CODE REVIEW

File: digitaltwins/service/src/service.json [hunk 1]
```diff
@@ -1536,7 +1781,7 @@
       "post": {
         "summary": "Invoke a digital twin interface command.",
         "description": "Invoke a digital twin interface command.",
-        "operationId": "DigitalTwin_InvokeInterfaceCommand",
+        "operationId": "DigitalTwin_InvokeComponentCommand",
```
Comment (anthonyvercolano) [inline digitaltwins/service/src/service.json:1784]: >InvokeComponentCommand" [](start = 36, length = 23)

This says component yet the parameter name is interfaceName, same for the path above.

File: digitaltwins/service/src/service.json [hunk 2]
```diff
@@ -2148,13 +2568,15 @@
             "TooManyModulesOnDevice",
             "ConfigurationCountLimitExceeded",
             "DigitalTwinModelCountLimitExceeded",
+            "InterfaceNameCompressionModelCountLimitExceeded",
```
Comment (anthonyvercolano) [inline digitaltwins/service/src/service.json:2571]: >InterfaceNameCompressionModelCountLimitExceeded [](start = 13, length = 47)

Pray what is a "Compression"Model?

**Comment by anthonyvercolano** [comment]
>    */

General comment: Why are we still supporting the callback signatures.  Why not just the promise?

---
Refers to: digitaltwins/service/src/cl/digital_twin_service_client.ts:148 in 4aadb60. [](commit_id = 4aadb60ec78cf01427c5ae335b8411a2e326a807, deletion_comment = False)

File: digitaltwins/service/src/service.json (hunk 1 above)
Comment (zolvarga) [reply digitaltwins/service/src/service.json:1784]: Yes, that is what has been generated. The service team still could not clean up this mess. I specifically asked them to get rid of all the "interfaces" but they probably have a reason.

---
In reply to: [449329625](https://github.com/Azure/azure-iot-sdk-node/pull/836#discussion_r449329625) [](ancestors = 449329625)

File: digitaltwins/service/src/service.json (hunk 2 above)
Comment (zolvarga) [reply digitaltwins/service/src/service.json:2571]: I pray with you...

---
In reply to: [449330230](https://github.com/Azure/azure-iot-sdk-node/pull/836#discussion_r449330230) [](ancestors = 449330230)

**Comment by zolvarga** [comment]
>    */

The questions is spot on. I actually was thinking to do that but I think for the preview is better to keep it. In the next check in I will clean up that.

---
In reply to: [653281582](https://github.com/Azure/azure-iot-sdk-node/pull/836#issuecomment-653281582) [](ancestors = 653281582)

---
Refers to: digitaltwins/service/src/cl/digital_twin_service_client.ts:148 in 4aadb60. [](commit_id = 4aadb60ec78cf01427c5ae335b8411a2e326a807, deletion_comment = False)

**Pull request closed without merging**